pkgs/tools/filesystems/android-file-transfer/default.nix

```diff
@@ -11,14 +21,24 @@ mkDerivation rec {
     sha256 = "125rq8ji83nw6chfw43i0h9c38hjqh1qjibb0gnf9wrigar9zc8b";
   };
 
-  nativeBuildInputs = [ cmake readline pkg-config ];
+  patches = lib.optional stdenv.isDarwin ./darwin-dont-vendor-dependencies.patch;
```

Comment:
Can we fetch the patch instead?

Reply:
I haven't created an upstream PR for this because it only solves a Nixpkgs packaging issue and won't benefit upstream.

Reply:
Can we apply the patch unconditional?

Reply:
Fixed.